src/Files.App/Views/Layouts/BaseGroupableLayoutPage.cs

```diff
@@ -252,7 +252,7 @@ protected virtual void StartRenameItem(string itemNameTextBox)
 
 			textBox.Focus(FocusState.Pointer);
 			textBox.LostFocus += RenameTextBox_LostFocus;
-			textBox.KeyDown += RenameTextBox_KeyDown;
+			textBox.PreviewKeyDown += RenameTextBox_KeyDown;
```

Comment:
Since the event is unsubscribed when the renaming is finished, it is necessary to change that to `PreviewKeyDown` as well.

Reply:
In theory, we should be able to check the logic for the Details View and copy from there.

Reply:
I pushed a commit with this change